src/components/content/content.ts

```diff
@@ -480,6 +482,18 @@ export class Content extends Ion implements OnDestroy, OnInit {
   }
 
   /**
+   * @input {boolean} If true, the content will scroll down on load.
+   */
+  @Input()
+  get scrollDownOnLoad(): boolean {
+    return !!this._scrollDownOnLoad;
```

Comment:
Sure, this can be removed if you'd like. I just copied the getter and setter from `@Input() fullscreen`.